h2o-algos/src/main/java/hex/glm/ComputationState.java

```diff
@@ -59,12 +62,21 @@ public ComputationState(Job job, GLMParameters parms, DataInfo dinfo, BetaConstr
     _nclasses = (parms._family == Family.multinomial||parms._family == Family.ordinal)?nclasses:1;
     _alpha = _parms._alpha[0];
   }
-
+  
   public GLMGradientSolver gslvr(){return _gslvr;}
   public double lambda(){return _lambda;}
   public void setLambdaMax(double lmax) {
     _lambdaMax = lmax;
   }
+  public void setActiveColsAll() {
+    _activeColsAll = new int[_nclasses][];
+    for (int classInd=0; classInd < _nclasses; classInd++) {
+      int[] original = activeDataMultinomial(classInd)._activeCols;
+      int activeColsLen = original.length;
+      _activeColsAll[classInd] = new int[activeColsLen];
+      System.arraycopy(original, 0, _activeColsAll[classInd], 0, activeColsLen);
+    }
+  }
```

Comment:
I don't see the advantage of copying the data. Is there any ? Can't we just link to it ? Correct me if I'm wrong please. Like that:
```suggestion
  public void setActiveColsAll() {
    _activeColsAll = new int[_nclasses][];
    for (int classInd=0; classInd < _nclasses; classInd++) {
      int[] original = activeDataMultinomial(classInd)._activeCols;
      _activeColsAll[classInd] = activeDataMultinomial(classInd)._activeCols;
    }
  }
```